Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.8
+
+RUN apk add --update ca-certificates \
+    && rm -rf /var/cache/apk/*
```

Comment:
You can also use `--no-cache` instead of `--update` and avoid the `rm -rf /var/cache/apk/` line. Feels cleaner.

Reply:
Does SIG RelEng want to document this or sort some tooling to have it consistent for all projects?

Reply:
I'm happy to write some docs in the SIG section of the giantswarm repo.

@xh3b4sd @MarcelMue @TheoBrigitte WDYT?

Reply:
I created a card on the releng board. If you want to initiate writing docs on this topic, then I am more then happy to review / help out.

Reply:
@MarcelMue Ack. Thx!